Services/Mail/classes/class.ilFormatMail.php

```diff
@@ -111,7 +111,7 @@ public function formatReplyRecipient()
     * @access	public
     * @return string
     */
-    public function formatForwardSubject()
+    public function formatForwardSubject() : mixed
```

Comment:
The `mixed` type is not supported in PHP 7.4, which we'll have to support as well in ILIAS 8.